### DISCUSSION TITLE
Async flow function

### DISCUSSION THREAD

**Discussion by punitarani** [microsoft/promptflow#1581] [Ideas]
I have created a flow and loaded it as a function similar to [get-started/flow-as-function](https://github.com/microsoft/promptflow/blob/1324b13c57941fb34c0e4194873c0742a0a0618c/examples/tutorials/get-started/flow-as-function.ipynb).

```python
f = load_flow(...)
output = f(input=input)
```

### Is there a way to load flow as an async function?

I want to run multiple flows in parallel and asynchronously.

```python
output = await f(input=input)
```

Is this possible?

**Comment by brynn-code** [comment]
Hi, I've created a feature ask issue for this :), please track here #1760
